wurst/systems/ui/Status.wurst

```diff
@@ -0,0 +1,18 @@
+package Status
+
+function moveStatusFrame()
+    // Assign the origin frame of the buff bar to a variable
+    let buffs = getOriginFrame(ORIGIN_FRAME_UNIT_PANEL_BUFF_BAR)
+    // Assign the origin frame label of the buff bar to a variable
+    let bufflabel = getOriginFrame(ORIGIN_FRAME_UNIT_PANEL_BUFF_BAR_LABEL)
+
+    // Clear the Points that link the label with the bar
+    bufflabel.clearAllPoints()
+
+    // We increase the scale and set the point to be just above the portrait
+    buffs.setScale(2)
```

Comment:
Can you use the `..` notation to condense calls in this file? E.g.

    buffs
        ..setScale(2)
        ..clearAllPoints()
        ..setAbsPoint(FRAMEPOINT_LEFT, 0.22, 0.16)